### ISSUE TITLE
Cannot install fixtures

### ISSUE THREAD

**Issue by minthemiddle** [volunteer-planner/volunteer_planner#13]
When trying to install fixture via `./manlocal.py loaddata vp_dumpdata_2015-09-08T08_37_28.266529_ANYNOMIZED.json` I get an error:

```
Models aren't loaded yet. (...)
`django.db.utils.IntegrityError: Problem installing fixture '/Users/martin/Coding/vp-official/vp_dumpdata_2015-09-08T08_37_28.266529_ANYNOMIZED.json': Could not load contenttypes.ContentType(pk=15): (1062, "Duplicate entry 'blueprint-blueprintcreator' for key 'django_content_type_app_label_45f3b1d93ec8c61c_uniq'")
```

I tried to run `migrate` again, but to no extend.

**Comment by Doca** [comment]
also if you have empty database? Thanks!

**Comment by minthemiddle** [comment]
Yes, even after i did `./manlocal.py flush` I get that error.

**Comment by christophmeissner** [comment]
Fixed with [Pull Request #14](https://github.com/volunteer-planner/volunteer_planner/pull/14)
